nodejs/examples/vpc/index.ts

```diff
@@ -53,4 +53,14 @@ const vpcWithLocations = new awsx.ec2.Vpc("custom6", {
         { type: "isolated", name: "db", location: "10.0.2.0/24" },
         { type: "isolated", name: "redis", location: "10.0.3.0/24" },
     ],
-}, providerOpts);
\ No newline at end of file
+}, providerOpts);
+
+const vpcWithSpecificZones = new awsx.ec2.Vpc("custom7", {
+    requestedAvailabilityZones: ["us-east-1a", "us-east-1b", "us-east-1c"],
+    subnets: [
+        { type: "public" },
+        { type: "private" },
+        { type: "isolated", name: "db" },
+        { type: "isolated", name: "redis" },
+    ],
+}, { provider: new aws.Provider("prov2", { region: "us-east-1" }) });
```

Comment:
I actually don't think there's a need to allocate a different provider here, so you should be able to just use `providerOpts` as with the other cases?

Reply:
@leezen 
Well ok, but I guess I gotta be asking for available zones and shuffle/pick since the tests are using env variables for region and I can't hard code them since those could change. 

Fixed in https://github.com/pulumi/pulumi-awsx/pull/638/commits/3eaa387353d1e216ad92afe18d426843ab1d62f8